rust/tests/tests.rs

```diff
@@ -194,6 +194,14 @@ mod tests {
         }
     }
 
+    #[test]
+    fn get_all_items_empty() {
+        let dict = dictionary::Dictionary::new("test_data/empty-key-dict.kv").unwrap();
```

Comment:
just leaving a code comment here, reflecting that this tests BWC

```suggestion
        // uses dictionary created using keyvi < 0.7.2, see gh#368
        let dict = dictionary::Dictionary::new("test_data/empty-key-dict.kv").unwrap();
```